serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -135,83 +153,68 @@ Learn how to use {serverless_logic_web_tools_name} for creating and managing wor
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-openapi-based-services.adoc[Orchestrating the OpenAPI services]
+xref:service-orchestration/orchestration-of-openapi-based-services.adoc[]
 [.card-description]
 Learn how to orchestrate REST services using OpenAPI specification descriptors
 --
 
 [.card]
 --
 [.card-title]
-xref:service-orchestration/orchestration-of-asyncapi-based-services.adoc[Orchestrating the AsyncAPI services]
-[.card-description]
-Learn how to trigger and consume events using AsyncAPI specification descriptors
---
-[.card]
---
-[.card-title]
-xref:service-orchestration/configuring-openapi-services-endpoints.adoc[Configuring the OpenAPI services endpoints]
+xref:service-orchestration/configuring-openapi-services-endpoints.adoc[]
```

Comment:
Same goes here. I would remove this card from the index. This is a "second level" card and I think it does not need to appear here,.

Reply:
Being in a second level does not mean not having it here. But I agree, we can remove it and keep the index simpler in this case since this could be a "next step" case.